aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/TagHelpers/AddressStyleTagHelperComponent.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace RazorPagesSample.TagHelpers
+{
+    #region snippet_AddressStyleTagHelperComponentClass
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Razor.TagHelpers;
+
+    public class AddressStyleTagHelperComponent : ITagHelperComponent
+    {
+        private readonly string _style = 
+            @"<link rel=""stylesheet"" href=""/css/address.css"" />";
```

Comment:
Earlier, I think you escaped a quote in a string in a Tag Helper Component, but you use the verbatim string literal here. Maybe go one way across the board?